documentation/src/main/asciidoc/configuration.asciidoc

```diff
@@ -1406,8 +1406,10 @@ configuration.
 Hibernate Search is included in the WildFly application server, and since WildFly 10 the module
 is automatically activated (added to the classpath of your deployment) if you are using Hibernate ORM and have any indexed entities.
 
-Alternatively you can opt to use a different version of the module by downloading and unzipping a different
-moduleset and setting the `wildfly.jpa.hibernate.search.module` property in your `persistence.xml`.
+WildFly includes an older version of Hibernate Search; if you want to use that then you won't need to download anything else and you can stop reading this chapter; however you might miss out on several new features and improvements.
```

Comment:
Which version should I add? All versions of WildFly include some Hibernate Search version, and it will always be older than what the latest version of this documentation mentiones.

Reply:
I'd just go with the one we target in our tests. It only seems logical that older versions of WildFly ship with even older versions of Hibernate Search.